### PR TITLE
Fix/simplify application properties

### PR DESCRIPTION
**Pull request by lmoesle**
**Description**

Short description or comments

**Reference**

Issues #XXX

### CODE REVIEW

File: digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    userid_            varchar(255) not null,
+    primary key (id_)
+);
+
+create table dwf_process_instance_info
+(
+    id_                   varchar(36)  not null,
+    processdefinitionkey_ varchar(255) not null,
+    processname_          varchar(255) not null,
+    description_          varchar(255),
```
Comment (dominikhorn93) [inline digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql]: 2000

File: digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    userid_        varchar(255) not null,
+    primary key (id_)
+);
+
+create table dwf_task_info
+(
+    id_             varchar(36)  not null,
+    assignee_       varchar(255),
+    definitionname_ varchar(255) not null,
+    description_    varchar(255),
```
Comment (dominikhorn93) [inline digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql]: 2000

File: digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql (hunk 1 above)
Comment (zambrovski) [reply digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql]: in H2 it should not matter a lot

File: digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql (hunk 2 above)
Comment (zambrovski) [reply digiwf-engine/digiwf-engine-service/src/main/resources/db/migration/h2/V3__digiwf.sql]: in H2 it should not matter a lot

**Review by StephanStrehlerCGI — APPROVED** [review]

**Pull request merged**